helm-deploy-test/tasks/cf-teardown.sh

```diff
@@ -1,17 +1,14 @@
 #!/bin/bash
-
 set -o errexit -o nounset
 
-# Export kube-host details from pool
+# Set kube config from pool
+mkdir /root/.kube/ && cp  pool.kube-hosts/metadata /root/.kube/config
+
 set -o allexport
-source pool.kube-hosts/metadata
-CF_NAMESPACE=cf
+CF_NAMESPACE=scf
```

Comment:
case scf wiki states it as scf. parity between manual and ci deployments

Reply:
That means we won't find bugs if they _don't_ match 😛

Reply:
:D no, I am testing it now

Reply:
looks okay https://ci.from-the.cloud/teams/main/pipelines/prabal-helm-deploy/jobs/helm-deploy-test/builds/6

Reply:
Right, what I mean is, if we had a bug in our code (say, we hard coded `scf` somewhere instead of grabbing the value from our namespace), making this match would mean we won't catch the problem in CI.

I guess that really means this should be `CF_NAMESPACE=scf-$(head -c3 /dev/urandom | base64)`? Random every time! XD

Reply:
I can revert to cf if u like. The problem with random namespace is that after setting a namespace in cf-deploy.sh, I will have to grep for the namespace via kubectl for teardown and run-tests scripts. I don't want to do that unless we benefit from random namespace. We can achieve what u suggest by using "cf" as namespace instead of scf.

Reply:
Ah, good point.
We can revert if you want to… but it sounds like neither one of us care that much, so merge it is.